metricbeat/etc/fields.yml

```diff
@@ -488,4 +488,91 @@ system:
               type: float
               description: >
                 The percentage of used swap memory.
+zookeeper:
+  type: group
+  description: >
+    Zookeeper metrics collected by the four-letter monitoring commands
+  fields:
+    - name: zookeeper-mntr
+      type: group
+      description: >
+        `zookeeper-mntr` contains the metrics reported by the four-letter `mntr`
+        command.
+      fields:
+        - name: hostname
+          type: keyword
+          description: >
+            hostname
+        - name: approximate_data_size
+          type: integer
+          description: >
+            Approximate size of zookeeper data
+        - name: avg_latency
+          type: integer
+          description: >
+            Average latency between ensemble hosts in milliseconds
```

Comment:
We usually end these description with dots. It's relevant because they are copied into the docs. The metricbeat `fields.yml` is not consistent about this atm.